### ISSUE TITLE
Not working with android

### ISSUE THREAD

**Issue by coolprobn** [ak1394/react-native-login#5]
I followed your tutorial and was able to integrate this into my project.
```
Login.start(config).then(tokens => {
  console.log(tokens);
});
```
Does not somehow work with android, as code never reaches then part, we have to handle incoming url manually.
Handling incoming deeplinks,
I am also able to get two codes in url under key: `code` and `state`.

How are these code supposed to be used further?

I want to extract access_token, refresh_token,user id and other informations

**Comment by ChenWes** [comment]
yes , check my project , still have error , not working with android
